crates/bevy_text/src/text2d.rs

```diff
@@ -168,17 +160,14 @@ pub fn update_text2d_layout(
         Changed<Text>,
         &Text,
         Option<&Text2dBounds>,
-        &mut Text2dSize,
         Option<&mut TextLayoutInfo>,
```

Comment:
This query now matches the non `Text2d` texts and breaks their layout.

Reply:
Ah didn't consider that.
A `Without<Node>` filter should do it?